docs/modules/ROOT/pages/api/finance.adoc

```diff
@@ -0,0 +1,296 @@
+:github-icon: pass:[<svg class="icon"><use href="#github-icon"/></svg>]
+:vesting-component: xref:VestingComponent[Vesting Component]
+:ownable-component: xref:api/access.adoc#OwnableComponent[Ownable Component]
+:vesting-schedule: xref:api/finance.adoc#VestingComponent-Vesting-Schedule[Vesting Schedule]
+:AmountReleased: xref:IVesting-AmountReleased[AmountReleased]
+
+= Finance
+
+This crate includes primitives for financial systems.
+
+== Vesting
+
+[.contract]
+[[IVesting]]
+=== `++IVesting++` link:https://github.com/OpenZeppelin/cairo-contracts/blob/release-v0.17.0/packages/finance/src/vesting/interface.cairo[{github-icon},role=heading-link]
+
+[.hljs-theme-dark]
+```cairo
+use openzeppelin_finance::vesting::interface::IVesting;
+```
+
```

Comment:
```suggestion

Common interface for Vesting-enabled wallets.

```

For consistency.

It would be good to add captions to the Timelock entries in the Governance API Reference too cc @andrew-fleming. Is not too important, but we should be as consistent as possbile.

Reply:
Done! #1167